Mathlib/RingTheory/Algebraic.lean

```diff
@@ -149,10 +149,17 @@ protected theorem IsAlgebraic.algHom {B} [Ring B] [Algebra R B] (f : A →ₐ[R]
   ⟨p, hp, by rw [aeval_algHom, f.comp_apply, ha, map_zero]⟩
 #align is_algebraic_alg_hom_of_is_algebraic IsAlgebraic.algHom
 
+theorem AlgHom.isAlgebraic_of_injective {B} [Ring B] [Algebra R B] (f : A →ₐ[R] B)
```

Comment:
```suggestion
theorem AlgHom.isAlgebraic_of_injective {B} [Ring B] [Algebra R B] {f : A →ₐ[R] B}
```

Reply:
I managed to sneak it in as [Algebra.IsAlgebraic.of_injective](https://github.com/leanprover-community/mathlib4/blob/05ad6c6d5e122ac4f6b3929b49039a0c1a646bab/Mathlib/RingTheory/Algebraic.lean#L162-L164) in #8406 😄 A common use case is `f := Subalgebra.val _` but `hf := Subtype.val_injective`, in which case Lean can't infer `f` from `hf`, so I'd like to keep it as is.